File: javascript/downloadPDF.js
const axios = require('axios').default;
const fs = require('fs');
const { argv } = require('process');
const command = require("./command");
// http://gazevedo996.pythonanywhere.com/
// "http://localhost:6546/"

const getdata = async () => {
    const url = "https://esaj.tjsp.jus.br/cdje/getPaginaDoDiario.do?cdVolume=18&nuDiario=3888&cdCaderno=10&nuSeqpagina=1&uuidCaptcha="    const response = await axios.get(url, {
        responseType: 'arraybuffer',
        headers: {
          "Access-Control-Allow-Origin": "*",
          "Access-Control-Allow-Methods": "GET,PUT,POST,DELETE,PATCH,OPTIONS"
        }
      });
    const pdfcontents = response.data;
    console.log(response)
    await fs.writeFile('file.pdf', pdfcontents, function(err) {
                if(err) {
                return console.log(err);
                }
                console.log("The file was saved!");
                });

    }
getdata();

File: download_pdf.py
from pathlib import Path
import requests
import sys
from string_bt import *
 
string_args=cat_string_list(sys.argv)
url, path = cat_arg("-u", string_args), cat_arg("-p", string_args)
filename = Path(path)
response = requests.get(url)
filename.write_bytes(response.content)
#print(response.content)
print(filename)
#Exemplo:
#cmd => py download_pdf.py -u "https://dje.tjsp.jus.br/cdje/getPaginaDoDiario.do?cdVolume=18&nuDiario=3912&cdCaderno=11&nuSeqpagina=263&uuidCaptcha=" -p "./diario2.pdf"
